### ISSUE TITLE
Vite/Vue: provide home page with minimal css

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/jhipster-lite#721]
The Vite/Vue context should contain 2 apis:
- one with empty home page, without CSS (what it is currently)
- another one with minimal css (similar to what is generated by vite+vue)

The 1st one can be used for starting real project
The 2nd one is used for demo, as it's better to have minimal CSS than a simple text

**Comment by matthieulapatate** [comment]
For now, only the files responsibles for starting the project have been created, so we just need to generate them throught APIs right ?

**Comment by pascalgrimaud** [comment]
yes, the new API with new methods will only generate another index.html file

**Comment by matthieulapatate** [comment]
Currently in process, soon available.
